### PR TITLE
Add .gitignore creation to hz init

### PR DESCRIPTION
**Pull request by dalanmiller**
Satisfies #454 

This change is [Reviewable](https://reviewable.io/reviews/rethinkdb/horizon/681)

### CODE REVIEW

File: cli/src/init.js [hunk 1]
```diff
@@ -228,6 +228,21 @@ function populateDir(projectName, dirWasPopulated, chdirTo, dirName) {
     // read/write only
   };
 
+  // Create .gitignore if it doesn't exist
+  if (!fileExists('.gitignore')) {
+    fs.appendFileSync(
+      '.gitignore',
+      `rethinkdb_data
+      **/*.log
```
Comment (deontologician) [inline cli/src/init.js]: What generates logs?

Comment (dalanmiller) [inline cli/src/init.js]: `npm` sometimes I believe? Specifically, npm-debug.log I've seen before.

**Comment by deontologician** [comment]
:lgtm:

**Comment by encryptio** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

_[cli/src/init.js, line 238 [r1]](https://reviewable.io:443/reviews/rethinkdb/horizon/681#-KNnBsg9thXosmsJNrhk:-KNnBsg9thXosmsJNrhl:-1113629036) ([raw file](https://github.com/rethinkdb/horizon/blob/fff396d6f489af9985997288feae96cc9619a051/cli/src/init.js#L238)):_

> ``` JavaScript
>       **/*.log
>       .hz/secrets.toml
>       `,
> ```

This will add leading spaces to all but the first line, due to the indentation inside the backticks.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/rethinkdb/horizon/681)_

**Comment by deontologician** [comment]
Made slight indentation change

**Comment by dalanmiller** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

_[cli/src/init.js, line 238 [r1]](https://reviewable.io:443/reviews/rethinkdb/horizon/681#-KNnBsg9thXosmsJNrhk:-KNncbunpa7jiXy6teJW:-499146009) ([raw file](https://github.com/rethinkdb/horizon/blob/fff396d6f489af9985997288feae96cc9619a051/cli/src/init.js#L238)):_
<details><summary>

<i>Previously, encryptio (Chris Kastorff) wrote…</i></summary>



> This will add leading spaces to all but the first line, due to the indentation inside the backticks.
> </details>
> Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/rethinkdb/horizon/681)_

**Comment by dalanmiller** [comment]
👍

**Comment by encryptio** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/rethinkdb/horizon/681#-:-KNne36cfUIPyy4uZ18W:1417088721)_

**Pull request merged**